src/main/java/org/broadinstitute/hellbender/tools/copynumber/PreprocessIntervals.java

```diff
@@ -99,6 +119,8 @@
 public final class PreprocessIntervals extends GATKTool {
     public static final String BIN_LENGTH_LONG_NAME = "bin-length";
     public static final String PADDING_LONG_NAME = "padding";
+    public static final String GRID_LONG_NAME = "grid";
+    public static final String MINIMUM_BIN_LENGTH_LONG_NAME = "min-bin-length";
```

Comment:
CNV argument names tend to be verbose, so this should be "minimum-bin-length".  I think the only exceptions where "min" and "max" are used are the gCNV inference parameters.